### ISSUE TITLE
contact field (email) on profiles. 

### ISSUE THREAD

**Issue by ken-spencer** [thaumazo/www.thaumazo.org#17]
Looking at the contact field under people it currently holds a single email address. A few questions comes to mind

- Would email: be a clearer name? 
- What is the intent in this being an array, anticipating more than one email address? 
- Potential concerns about displaying a person's contact information.  This could result in spam harvesting, and bad actors getting internal emails.  I would recommend putting a contact form on a the profile page for reaching someone directly. 
- if we want to use the email icon (as per figma) we could use it link to a form (or launch a form modal).

**Comment by DanielThaumazo** [comment]
Agreed re: displaying contact info. I believe array is unnecessary.

**Comment by ken-spencer** [comment]
May obfuscate them some to make harder to scrape from the public github.
foo at bar dot com
Or something similar.  Keeping the contact field might actually make them harder to scrape as well.
